lib/packages/core/lib/src/blocs/participants/handler/get_participants_handler.dart

```diff
@@ -0,0 +1,49 @@
+part of "../participants_bloc.dart";
+
+void _getParticipantsEventHandler(LMChatGetParticipantsEvent event,
+    Emitter<LMChatParticipantsState> emit) async {
+  emit(const LMChatParticipantsLoadingState());
+  try {
+    final GetParticipantsRequest getParticipantsRequest =
```

Comment:
Try to add comments here as well, if possible.

Reply:
done